src/Service/Timestamper.php

```diff
@@ -33,13 +30,8 @@ public function add(TimestampableEntityInterface $entity, DateTime $timestamp)
         if (!array_key_exists($class, $this->entities[$ts])) {
             $this->entities[$ts][$class] = [];
         }
-        if (!$entity instanceof IdentifiableEntityInterface) {
-            throw new Exception("Tried to timestamp a non identifiable entity {$class}");
-        }
         // When and entity has already been deleted it will lose it's ID so we have to check for that here
```

Comment:
this comment should probably go if we don't check for the existence of the entity id anymore.

Reply:
✅ I updated it to make sense.